src/os/CMakeLists.txt

```diff
@@ -115,16 +121,22 @@ endif()
 
 if(WITH_SPDK)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
+  add_custom_target(build_dpdk
+    COMMAND
+    make config T=x86_64-native-linuxapp-gcc && make EXTRA_CFLAGS="-fPIC"
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src/spdk/dpdk
+    COMMENT "dpdk building")
```

Comment:
yes... we should make provisions for that? I am not an expert about this, could you give me some guidance? Thanks!